### PR TITLE
Provide backend methods to update the pw of current user

### PR DESCRIPTION
**Pull request by buehner**
Title says it all.

Related:

https://github.com/terrestris/momo3-admin/pull/69
https://github.com/terrestris/momo3-login/pull/10

### CODE REVIEW

File: src/main/java/de/terrestris/momo/web/MomoUserController.java [hunk 1]
```diff
@@ -177,6 +177,31 @@ public void activateMomoUser(
...
+
+		LOG.debug("Requested to update a password.");
+
+		try {
+			boolean success = this.service.updatePassword(oldPassword, newPassword);
+			if(success) {
+				return ResultSet.success("Password has successfully been updated.");
+			}
+			return ResultSet
+					.error("Password could NOT been updated. Please check if the old password was passed correctly.");
```
Comment (marcjansen) [inline src/main/java/de/terrestris/momo/web/MomoUserController.java]: ... could not ~been~ *be* updated

**Review by marcjansen — APPROVED** [review]
One comment, looks nice otherwise. Please  erge at will

**Comment by marcjansen** [comment]
Service-wise this allows a user to have the empty string as password, right? we should probably have some minimum standards for passwords, do you agree?

**Comment by buehner** [comment]
:+1: @marcjansen i'll address your issues tomorrow. thx for the review

**Comment by weskamm** [comment]
please remember the registration form and password forgotten form in the `login` repo when implementing `minimum standards`

**Comment by weskamm** [comment]
premium, good to go

**Pull request merged**